### ISSUE TITLE
Transparency issue after showing menu bar and hiding it

### ISSUE THREAD

**Issue by subhasutra** [Bali10050/Darkly#82]
In dolphin showing menubar and then hiding it is causing a translucent gap in the place of the menubar. Happens with gwenview as well among the other kde apps.

![dolphin](https://github.com/user-attachments/assets/9394adb7-b5dc-4cb4-aca9-b6c3a2bb4954)
![dolphin_withmenubar](https://github.com/user-attachments/assets/d8c0cec2-98e8-4698-810f-15de58aa28ff)
![dolphin_hidemenubar](https://github.com/user-attachments/assets/7e904425-96d6-4450-8f21-408d1802c8f0)

**Comment by DeltaCopy** [comment]
Is this happening on all color schemes or just one in particular ?

**Comment by DeltaCopy** [comment]
#90 fix maybe ?

**Comment by subhasutra** [comment]
Yes that fixes the issue. But it adds some flicker, although I doubt anyone will use it that way.


https://github.com/user-attachments/assets/f99d8fbe-a15e-4af4-ac21-81331e750078
